tests/fixture/tmpnet/utils.go

```diff
@@ -87,3 +87,12 @@ func NewPrivateKeys(keyCount int) ([]*secp256k1.PrivateKey, error) {
 	}
 	return keys, nil
 }
+
+func GetVMID(vmName string) (ids.ID, error) {
```

Comment:
If ANR does like that it's fine with me. That is actually a user facing product so it's the most informative I guess

Reply:
I've always hated this. This is really not what VMIDs are supposed to be at all... it's just an ugly hack because it happens to work with the VMs we've developed internally.

It is not possible to determine the VMID from the name of the VM (or chain, or subnet). Which is why I didn't allow this function into avalanchego's utils.

Reply:
> How should it be determined?

Currently VMs arbitrarily specify their `VMID`s. (so I think the user of `tmpnet.Chain` should just provide the `VMID`.

My hope in the future is that `VMID`s will come from the public key of the developer of the VM or something which will fit nicely into a VM marketplace.

But I think rather than specifying names it should be expected to specify the ID.